api/lua/Cargo.toml

```
@@ -0,0 +1,67 @@
+[package]
+name = "playdate-lua"
+version = "0.1.0"
+readme = "README.md"
+description = "High-level Lua API built on-top of Playdate API"
+keywords = ["playdate", "sdk", "api", "gamedev"]
+categories = ["game-development", "api-bindings", "no-std"]
+edition.workspace = true
+license.workspace = true
+authors = ["Paul Young <84700+paulyoung@users.noreply.github.com>", "Alex Koz <a@koz.email>"]
+homepage.workspace = true
+repository.workspace = true
+
+
+[features]
+default = ["sys/default"]
+
+# playdate-sys features, should be shared because it's build configuration:
+
+bindgen-runtime = ["sys/bindgen-runtime"]
+bindgen-static = ["sys/bindgen-static"]
+bindings-derive-debug = ["sys/bindings-derive-debug"]
+
+
+[dependencies]
+sys = { workspace = true, default-features = false }
+
+[dev-dependencies]
+system = { workspace = true, default-features = false, features = [ "try-trait-v2" ] }
```

```suggestion
system = { workspace = true, default-features = false }
```

Feature `try-trait-v2` needed only if you want to do implicit convention `Result -> EventLoopCtrl` as well as to `UpdateCtrl` and use try sugar `?` with it. 
This is not used in your example, so no need to enable the feature.